### ISSUE TITLE
Command to install does not work on Mac (M2 chip)

### ISSUE THREAD

**Issue by cossio** [tectonic-typesetting/tectonic#943]
I tried running the following command on a MacBook (M2 chip):

```
curl --proto '=https' --tlsv1.2 -fsSL https://drop-sh.fullyjustified.net |sh
```

It gives the following error:

```
tectonic(drop-installer.sh): unknown CPU type: arm64
```

Is there any other way to install?

**Comment by pkgw** [comment]
For Macs on ARM chips ("Apple Metal", right?), you have to [compile it yourself][1], because we don't yet have CI builds for that platform. I've started playing around with seeing if I can get a build going because it would definitely be nice to have, but we use Azure Pipelines for CI and they don't have Metal native hosts (I think??) so it will probably take a while to get anything going.

[1]: https://tectonic-typesetting.github.io/book/latest/howto/build-tectonic/index.html

**Comment by pkgw** [comment]
In #959 I've prototyped a CI job to create a Tectonic build for Apple Metal. Would anyone be able to give this file a spin and see if it works? Thanks!

https://newton.cx/~peter/files/tectonic-0.11.0+20221025-aarch64-apple-darwin.tar.gz

**Comment by pkgw** [comment]
OK, hopefully this is fixed with the 0.12 release and tectonic-typesetting/tectonic-typesetting.github.io#44!

**Comment by pkgw** [comment]
Thanks! If anyone can try the `curl -fsSL https://drop-sh.fullyjustified.net/ |sh` installer on Apple Silicon, I think that should work now too, but I don't think I've seen any confirmation of that.
